docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -223,9 +237,16 @@ public StartContainerCmd createHostConfig(StartContainerCmd hostConfig) {
         if( dnsHosts.length > 0 )
             hostConfig.withDns(dnsHosts);
 
-        // ?
-        //if (volumes.length > 0)
-        //    hostConfig.with VolumesFrom (volumes);
+        if(volumesString!=null) {
+            for (String volume : splitAndFilterEmpty(volumesString)) {
+                try {
+                    hostConfig.withBinds(Bind.parse(volume));
+                } catch (IllegalArgumentException e) {
+                    LOGGER.log(Level.WARNING,"Cannot create volume "+volume+" when starting host");
+                    //hostConfig.withVolumes(Volume.parse(volume));
```

Comment:
In fact there is no withVolumes for `StartContainerCmd` and almost all methods of it have been made deprecated:
'TODO: Almost all methods are deprecated as they have corresponding siblings in {@link CreateContainerCmd} now.'
from https://github.com/docker-java/docker-java/blob/master/src/main/java/com/github/dockerjava/api/command/StartContainerCmd.java
I've tried to be as consistent as possible from the existing code